src/python/espressomd/reaction_ensemble.pyx

```diff
@@ -35,6 +35,8 @@ cdef class ReactionAlgorithm:
         each other. The Boltzmann factor :math:`\\exp(-\\beta E)` gives these
         configurations a small contribution to the partition function,
         therefore they can be neglected.
+    seed : :obj:`int`
+        Initial counter value (or seed) of the philox RNG.
```

Comment:
```suggestion
        Initial counter value (or seed) of the Mersenne Twister RNG.
```
Source code of `class ReactionAlgorithm`:
https://github.com/espressomd/espresso/blob/72da9ac0f41486e51c0882d171cdf6ab8e797787/src/core/reaction_ensemble.hpp#L191-L192